app/helpers/metric_item_helper.rb

```diff
@@ -1,6 +1,10 @@
 module MetricItemHelper
-  def metric_item(identifier, sampled: false, html: {})
-    item = MetricItem.new(self)
+  include GovernmentServiceDataAPI::MetricStatus
+
+  def metric_item(identifier, sampled: false, metric_value: nil, html: {})
```

Comment:
Is `metric_value` ever nil? As far as I can see it's always passed in.

Reply:
Yes, it is now always passed in, Think it's from when I was doing templates one at a time. Will remove.